coalib/results/Diff.py

```diff
@@ -78,7 +78,7 @@ def from_clang_fixit(cls, fixit, file):
                     fixit.value +
                     oldvalue[endindex:])
         new_file = (file[:fixit.range.start.line-1] +
-                    newvalue.splitlines(True) +
+                    type(file)(newvalue.splitlines(True)) +
                     file[fixit.range.end.line:])
```

Comment:
I think it may be better to convert file to list. In this way we'll be sure of what type is new_file. Also, some issues may arise if type of file will be something obscure, like neither list nor tuple.